roles/sft-monitoring-certs/tasks/ca-cert.yml

```diff
@@ -17,7 +17,6 @@
     path: "{{ workspace.path }}/ca-cert.pem"
     privatekey_path: "{{ workspace.path }}/ca-key.pem"
     csr_path: "{{ workspace.path }}/ca-csr.pem"
-    selfsigned_not_after: "+3650d"
```

Comment:
You mentioned the 10 years. Would it make sense to mention this in a comment here somewhere?

Reply:
Added the comment.